src/components/features/about-section/components/about-content.tsx

```
@@ -0,0 +1,35 @@
+import Subtitle from "@components/shared/subtitle";
+import DiscoverButton from "./discover-button";
+import AboutFeatures from "./about-features";
+
+export default function AboutContent() {
+    return (
+        <div className="flex-1">
```

ESLint

[error] className value must use cn() function. Example: className={cn("...")}

(local-rules/require-cn-classname)